soc/nordic/common/dmm.h

```diff
@@ -34,7 +34,9 @@ extern "C" {
  * Cache line alignment is required if region is cacheable and data cache is enabled.
  */
 #define DMM_REG_ALIGN_SIZE(node_id) \
-	(DMM_IS_REG_CACHEABLE(node_id) ? CONFIG_DCACHE_LINE_SIZE : sizeof(uint8_t))
+	COND_CODE_0(CONFIG_DCACHE,  \
```

Comment:
i don't think that this will work. `CONFIG_DCACHE` or exists and is set to 1 or is not defined. `COND_CODE_0` will evaluate to first option only if `CONFIG_DCACHE` is defined as 0. Use `COND_CODE_1` and swap options.

Reply:
Thanks, I keep forgetting this. Updated.